tests/bluetooth/tester/src/audio/btp_bap_broadcast.c

```diff
@@ -484,13 +487,24 @@ uint8_t btp_bap_broadcast_source_stop(const void *cmd, uint16_t cmd_len,
 
 	LOG_DBG("");
 
+	k_sem_reset(&sem_stream_stopped);
+
 	err = bt_bap_broadcast_source_stop(source->bap_broadcast);
 	if (err != 0) {
 		LOG_DBG("Unable to stop broadcast source: %d", err);
 
 		return BTP_STATUS_FAILED;
 	}
 
+	for (int i = 0; i < ARRAY_SIZE(source->streams); i++) {
```

Comment:
this should check for actual number of streams, not max supported, shouldn't it?

Reply:
Right, must this be done with stream->in_use, or is this number stored in the struct?

Reply:
> Right, must this be done by checking stream->in_use, or is this number stored somewhere in the source struct?

I don't think we currently store it anywhere, but it's pretty easy to just add a counter in the `struct btp_bap_broadcast_remote_source` so that we always know how many `sem_stream_stopped` we should expect if we ever start fewer than `ARRAY_SIZE(source->streams)` streams

@sjanc any input?

Reply:
`stream_stopped` will be called per stream, regardless of the number of subgroups